libp2p/NodeTable.cpp

```diff
@@ -621,36 +645,52 @@ void NodeTable::onPacketReceived(
 
 void NodeTable::doDiscovery()
 {
-    m_timers.schedule(c_bucketRefresh.count(), [this](boost::system::error_code const& _ec)
-    {
-        if (_ec)
-            // we can't use m_logger here, because captured this might be already destroyed
-            clog(VerbosityDebug, "discov")
-                << "Discovery timer was probably cancelled: " << _ec.value() << " "
-                << _ec.message();
-
-        if (_ec.value() == boost::asio::error::operation_aborted || m_timers.isStopped())
+    m_discoveryTimer.expires_from_now(boost::posix_time::milliseconds(c_bucketRefresh.count()));
```

Comment:
I think you can define the constant now as `boost::posix_time::milliseconds`

Reply:
I'm going to use steady_timer so we can use std::chrono time values